### ISSUE TITLE
[mesheryctl] Unhandled error in `system start`

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#4037]
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior
mesheryctl panics when running `system start` using v0.5.51

#### Screenshots/Logs
```
➜  ~ mesheryctl system start

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x1482937]

goroutine 1 [running]:
github.com/layer5io/meshery/mesheryctl/internal/cli/root/config.(*MesheryCtlConfig).CheckIfCurrentContextIsValid(0x0, 0x39263e0, 0x2967ae1, 0x2)
	/Users/runner/work/meshery/meshery/mesheryctl/internal/cli/root/config/config.go:71 +0x37
github.com/layer5io/meshery/mesheryctl/internal/cli/root/config.(*MesheryCtlConfig).GetCurrentContext(0x0, 0x0, 0x0, 0x0)
	/Users/runner/work/meshery/meshery/mesheryctl/internal/cli/root/config/config.go:107 +0x2f
github.com/layer5io/meshery/mesheryctl/internal/cli/root/system.glob..func17(0x38fe8a0, 0x3ae0510, 0x0, 0x0, 0x0, 0x0)
	/Users/runner/work/meshery/meshery/mesheryctl/internal/cli/root/system/start.go:71 +0xb3
github.com/spf13/cobra.(*Command).execute(0x38fe8a0, 0x3ae0510, 0x0, 0x0, 0x38fe8a0, 0x3ae0510)
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:845 +0x523
github.com/spf13/cobra.(*Command).ExecuteC(0x38f9b20, 0x2c18280, 0x3ae0510, 0x0)
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/layer5io/meshery/mesheryctl/internal/cli/root.Execute()
	/Users/runner/work/meshery/meshery/mesheryctl/internal/cli/root/root.go:78 +0x56
main.main()
	/Users/runner/work/meshery/meshery/mesheryctl/cmd/mesheryctl/main.go:22 +0x25
```

#### Environment

- Host OS: Mac
- Meshery Server Version: stable-v0.5.51
- Meshery Client Version: stable-v0.5.51
- Platform: Docker

---

#### Contributor Resources
- [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

**Comment by san-coding** [comment]
Hey, I am new to open source, can you please guide me to fix this issue ?

**Comment by piyushsingariya** [comment]
@san-coding this might not be the best issue to start as a newcomer, let's stick to that Integration Test issue that has been assigned to you.

**Comment by panyuenlau** [comment]
I'd like to look into this, can I get assigned for this issue?

**Comment by piyushsingariya** [comment]
sure!

**Comment by panyuenlau** [comment]
Hey @piyushsingariya, I just looked into the implementation and it seems this PR https://github.com/meshery/meshery/pull/4035 fixed the panic in release v0.5.52? @Revolyssup @leecalcote can you please confirm if this is still a valid issue?
